app/src/main/java/com/amaze/filemanager/fragments/MainFragment.java

```diff
@@ -1594,18 +1597,30 @@ public String getCurrentPath() {
     private void addShortcut(LayoutElementParcelable path) {
         //Adding shortcut for MainActivity
         //on Home screen
-        Intent shortcutIntent = new Intent(getActivity().getApplicationContext(),
-                MainActivity.class);
+        final Context ctx = getActivity().getApplicationContext();
```

Comment:
Reason for not using getContext()?

Reply:
In fact, the reason was my intent to change as little as possible. It seems this really does not matter. I will change it to `getContext()`.